examples/0_Simple/vectorAdd_lib/host.gpr

```diff
@@ -0,0 +1,41 @@
+------------------------------------------------------------------------------
+--                       Copyright (C) 2017, AdaCore                        --
+-- This is free software;  you can redistribute it  and/or modify it  under --
+-- terms of the  GNU General Public License as published  by the Free Soft- --
+-- ware  Foundation;  either version 3,  or (at your option) any later ver- --
+-- sion.  This software is distributed in the hope  that it will be useful, --
+-- but WITHOUT ANY WARRANTY;  without even the implied warranty of MERCHAN- --
+-- TABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public --
+-- License for  more details.  You should have  received  a copy of the GNU --
+-- General  Public  License  distributed  with  this  software;   see  file --
+-- COPYING3.  If not, go to http://www.gnu.org/licenses for a complete copy --
+-- of the license.                                                          --
+------------------------------------------------------------------------------
+
+with "../../../api/cuda_host.gpr";
+
+project Host is
+
+   for Exec_Dir use ".";
+   for Object_Dir use "obj/host";
+   for Source_Dirs use ("src/common", "src/host");
+   for Main use ("main.adb");
+
+   for Create_Missing_Dirs use "True";
+
+   package Compiler is
+      for Switches ("ada") use ("-gnatX", "-g", "-gnatd_c", "-gnatDGL");
+   end Compiler;
+
+   package Linker is
+      for Switches ("ada") use ("-L/usr/lib/cuda/targets/x86_64-linux/lib/stubs", "-L/usr/lib/cuda/targets/x86_64-linux/lib", "-lcudadevrt", "-lcudart_static", "-lrt", "-lpthread", "-ldl",
+         "-Wl,--unresolved-symbols=ignore-all");
```

Comment:
I don't think `-Wl,--unresolved-symbols=ignore-all` is needed anymore, and if it is there's a problem :/
```suggestion
      for Switches ("ada") use ("-L/usr/lib/cuda/targets/x86_64-linux/lib/stubs", "-L/usr/lib/cuda/targets/x86_64-linux/lib", "-lcudadevrt", "-lcudart_static", "-lrt", "-lpthread", "-ldl");
```